---
title: "Genetic Tests"
---
# Introduction

I specialise in making new genetic tests for clinical laboratories. This page lists the tests I have developed so far in my career, starting with the most recent. All of these projects were collaborations with many wonderful colleagues who I am greatly indebted to.

I have included links to the Github code repositories that I wrote for each project.

# Deletions, LOH and Ploidy Detection in Solid Cancers

This project was the final section of a broader project to validate copy number variant detection for solid cancers at the North West GLH. It was the most technically complex validation I've done so far, involving over 150 samples, 47 different genes and 4 different laboratory methods, with investigation of gene deletions, loss of heterozygosity (LOH) and different chromosomal ploidy states. 

**Project Overview**

**Outcome:** diagnostic service

**Timing:** Nov 2024-April 2025

**Location:** Manchester Genomic Laboratory Hub

**Technique:** [QIAseq](https://pubmed.ncbi.nlm.nih.gov/28049435/) next generation sequencing

**Codebase:** [R and Quarto](https://github.com/joe-m-shaw/pansolid_cnv_validation)

**Skills I learned**

- How to create my first R package: [foldchanger](https://joe-m-shaw.github.io/foldchanger/)
- How to perform automated code testing with testthat
- How to perform validation comparisons across multiple techniques and data layouts, including whole genome sequencing, droplet digital PCR, short tandem repeat PCR and NGS

# Amplifications of 9 Oncogenes in Diverse Cancers

This project built on my previous validation of *ERBB2* amplifications, adding a further 8 genes: *ALK*, *BRAF*, *EGFR*, *MET*, *MDM2* *MYC*, *MYCN* and *SMO*. These genes are important for prognosis and treatment planning in many different cancer types. 

**Project Overview**

**Outcome:** diagnostic service

**Timing:** Apr 2024-Nov 2024

**Location:** Manchester Genomic Laboratory Hub

**Technique:** [QIAseq](https://pubmed.ncbi.nlm.nih.gov/28049435/) next generation sequencing

**Codebase:** [R](https://github.com/joe-m-shaw/pansolid_cnv_validation)

**Skills I learned**

- How to build a shiny app for quality monitoring
- How to render full documentation and analysis with R and Quarto into a Word template
- How to manipulate whole genome sequencing HTML results using rvest

# *ERBB2* Amplifications in Solid Cancers

This project was to detect amplifications of the *ERBB2* (*HER2*) gene in solid tumour samples, focusing on lung, colorectal and brain cancers. *ERBB2* is an important gene in many cancer types, and is targeted by several medications including trastuzumab (Herceptin).

[**Project Overview**]{.underline}

**Outcome:** diagnostic service

**Timing:** Dec 2023-Apr 2024

**Location:** Manchester Genomic Laboratory Hub

**Technique:** [QIAseq](https://pubmed.ncbi.nlm.nih.gov/28049435/) next generation sequencing

**Codebase:** [R](https://github.com/joe-m-shaw/pansolid_cnv_validation)

[**Skills I learned**]{.underline}

- How to create CNV plots with embedded gene, exon and primer locations
- How to combine R and Quarto for analysis
- How to perform limit of detection experiments with reference materials

# Genomic Instability Testing for Ovarian Cancer

In this project I validated a new bioinformatics pipeline for detection of genomic instability in ovarian cancer, which was developed by the company [SeqOne Genomics](https://seqone.com/). Accurate detection of genomic instability is necessary for identifying optimum-responders to the anti-cancer drug olaparib (LynparzaÂ®). This workflow is now in diagnostic service, and the results are used to plan patient treatment.

[**Project Overview**]{.underline}

**Outcome:** diagnostic service

**Timing:** Sept 2023-Dec 2023

**Location:** Manchester Genomic Laboratory Hub

**Technique:** shallow whole genome sequencing (sWGS)

**Codebase:** [R](https://github.com/joe-m-shaw/homologous_recombination_deficiency)

[**Skills I learned**]{.underline}

- How to analyse sWGS data 
- How to query an SQL database with dbplyr using an ODBC connection
- How to parse information from PDFs with pdftools using regular expressions and raw strings

# Spinocerebellar Ataxia type 36

Spinocerebellar Ataxia type 36 ([SCA36](https://pubmed.ncbi.nlm.nih.gov/25593102/)) is a very rare neurological disorder caused by a hexanucleotide repeat expansion in the *NOP56* gene, with fewer than 200 patients reported in the literature. I optimised a repeat-primed PCR protocol and validated it on a cohort of samples tested via whole genome sequencing with the [ExpansionHunter](https://pubmed.ncbi.nlm.nih.gov/31134279/) algorithm. This diagnostic service is, to my knowledge, the first in the world for this condition.

[**Project Overview**]{.underline}

**Outcome:** diagnostic service

**Timing:** Nov 2022-July 2023

**Location:** University College London Hospital Neurogenetics Department

**Technique:** repeat-primed PCR and flanking PCR

**Codebase:** [R](https://github.com/joe-m-shaw/repeat_confirmations_project)

[**Skills I learned**]{.underline}

- How to design repeat-primed PCR primers
- How to optimise repeat-primed PCRs 
- How to integrate a literature review into a validation document

# *RFC1*-Disorder Testing

*RFC1*-disorder is a neurological disorder caused by repeat expansions in the *RFC1* gene, and was first [reported in 2019](https://pubmed.ncbi.nlm.nih.gov/30926972/) by Dr Andrea Cortese's research group at the UCLH Institute for Neurology. For this project I developed a diagnostic service for *RFC1* expansions by modifying the available research protocol, and successfully added *RFC1* testing to the [NHS National Genomic Test Directory](https://www.england.nhs.uk/publication/national-genomic-test-directories/).

[**Project Overview**]{.underline}

**Outcome:** diagnostic service

**Timing:** Jan 2022-July 2023

**Location:** University College London Hospital Neurogenetics Department

**Technique:** multiple repeat-primed PCRs and flanking PCR

**Codebase:** [R](https://github.com/joe-m-shaw/repeat_confirmations_project)

[**Skills I learned**]{.underline}

- How to fully optimise a flanking PCR to detect large amplicons
- How to apply to the National Genomic Test Directory for new test indication
- How to integrate multiple PCRs into a single workflow for increased efficiency and cost-effectiveness

# Mosaicism Detection for Rare Skin Disorders

The aim of this project was to confirm mosaic sequence variants for rare skin disorders using droplet digital PCR (ddPCR). The main challenges were that the variants were sometimes present in 1% of the sample or lower, the DNA quantities from skin biopsies were limited, and many of the variants were only found in a single patient. The solution was to use R to collate all the ddPCR data for multiple assays and establish uniform thresholds for background normal variation. The technique was then brought into diagnostic service, and over 100 reports were issued in the first 6 months.

[**Project Overview**]{.underline}

**Outcome:** diagnostic service

**Timing:** Oct 2021-March 2022

**Location:** North London Genomic Laboratory Hub

**Technique:** ddPCR

**Codebase:** [R](https://github.com/joe-m-shaw/ddpcr_mosaicism)

[**Skills I learned**]{.underline}

- How to set data-driven analysis thresholds for ddPCR
- How to use RMarkdown to generate validation plots and tables
- How to automatically audit clinical services using RMarkdown scripts

# Non-Invasive Prenatal Diagnosis for Sickle Cell Disease and Rare Genetic Conditions

In this project I developed a ddPCR test for sickle cell disease which could diagnose the condition prenatally using only a maternal blood sample. The project was then expanded to developing bespoke ddPCR assays for rare genetic conditions with different inheritance patterns. The correct prediction rate was optimised to 98% in comparison with invasive testing, but the presence of 2 incorrect results prevented the implementation of the method as a diagnostic service, and we [published our findings](https://pubmed.ncbi.nlm.nih.gov/36760169/) in the journal Prenatal Diagnosis.

[**Project Overview**]{.underline}

**Outcome:** publication

**Timing:** September 2019-January 2022

**Location:** North London Genomic Laboratory Hub

**Technique:** droplet digital PCR

**Codebase:** [R and JAGS](https://github.com/joe-m-shaw/ddpcr_nipd)

[**Skills I learned**]{.underline}

- How to write an original research paper
- How to write code in R using tidyverse packages
- How to optimise ddPCR assays

# Myotonic Dystrophy Type 1

This was my first validation project, which I began as a trainee clinical scientist. Myotonic dystrophy type 1 is caused by a triplet repeat expansion in the *DMPK* gene, and testing for this condition is a core requirement for genetics laboratories in England. I was asked to validate the test at the North London Genomic Laboratory Hub as part of my clinical scientist training. The validation successfully passed external evaluation by the United Kingdom Accreditation Service (UKAS) and went into diagnostic service. 

[**Project Overview**]{.underline}

**Outcome:** diagnostic service

**Timing:** July 2018-Jan 2020

**Location:** North London Genomic Laboratory Hub

**Technique:** repeat-primed PCR

[**Skills I learned**]{.underline}

- How to write a validation document to the standards of ISO:15189
- How to calculate mobility shifts in capillary electrophoresis using GeneMarker
- How to write standard operating procedures for wet lab work and analysis
