<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.433">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Joe Shaw - Genetic Tests</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Joe Shaw</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html" rel="" target="">
 <span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./projects.html" rel="" target="" aria-current="page">
 <span class="menu-text">Genetic Tests</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./publications.html" rel="" target="">
 <span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./education.html" rel="" target="">
 <span class="menu-text">Experience and Training</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./fun_projects.html" rel="" target="">
 <span class="menu-text">Fun Projects</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#genomic-instability-testing-for-ovarian-cancer" id="toc-genomic-instability-testing-for-ovarian-cancer" class="nav-link" data-scroll-target="#genomic-instability-testing-for-ovarian-cancer">Genomic Instability Testing for Ovarian Cancer</a></li>
  <li><a href="#spinocerebellar-ataxia-type-36" id="toc-spinocerebellar-ataxia-type-36" class="nav-link" data-scroll-target="#spinocerebellar-ataxia-type-36">Spinocerebellar Ataxia type 36</a></li>
  <li><a href="#rfc1-disorder-testing" id="toc-rfc1-disorder-testing" class="nav-link" data-scroll-target="#rfc1-disorder-testing"><em>RFC1</em>-Disorder Testing</a></li>
  <li><a href="#mosaicism-detection-for-rare-skin-disorders" id="toc-mosaicism-detection-for-rare-skin-disorders" class="nav-link" data-scroll-target="#mosaicism-detection-for-rare-skin-disorders">Mosaicism Detection for Rare Skin Disorders</a></li>
  <li><a href="#non-invasive-prenatal-diagnosis-for-sickle-cell-disease-and-rare-genetic-conditions" id="toc-non-invasive-prenatal-diagnosis-for-sickle-cell-disease-and-rare-genetic-conditions" class="nav-link" data-scroll-target="#non-invasive-prenatal-diagnosis-for-sickle-cell-disease-and-rare-genetic-conditions">Non-Invasive Prenatal Diagnosis for Sickle Cell Disease and Rare Genetic Conditions</a></li>
  <li><a href="#myotonic-dystrophy-type-1" id="toc-myotonic-dystrophy-type-1" class="nav-link" data-scroll-target="#myotonic-dystrophy-type-1">Myotonic Dystrophy Type 1</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Genetic Tests</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="introduction" class="level1">
<h1>Introduction</h1>
<p>I specialise in making new genetic tests for clinical laboratories. This page lists the tests I have developed so far in my career, starting with the most recent. All of these projects were collaborations with many wonderful colleagues who I am greatly indebted to.</p>
</section>
<section id="genomic-instability-testing-for-ovarian-cancer" class="level1">
<h1>Genomic Instability Testing for Ovarian Cancer</h1>
<p>In this project I validated a new bioinformatics pipeline for detection of genomic instability in ovarian cancer, which was developed by the company <a href="https://seqone.com/">SeqOne Genomics</a>. Accurate detection of genomic instability is necessary for identifying optimum-responders to the anti-cancer drug olaparib (Lynparza®). This workflow is now in diagnostic service, and the results are used to plan patient treatment.</p>
<p><u><strong>Project Overview</strong></u></p>
<p><strong>Outcome:</strong> diagnostic service</p>
<p><strong>Timing:</strong> Sept 2023-Dec 2023</p>
<p><strong>Location:</strong> Manchester Genomic Laboratory Hub</p>
<p><strong>Technique:</strong> shallow whole genome sequencing (sWGS)</p>
<p><strong>Codebase:</strong> <a href="https://github.com/joeshaw824/homologous_recombination_deficiency">R</a></p>
<p><u><strong>Skills I learned</strong></u></p>
<ul>
<li>How to analyse sWGS data</li>
<li>How to query an SQL database with dbplyr using an ODBC connection</li>
<li>How to parse information from PDFs using regular expressions and raw strings</li>
</ul>
</section>
<section id="spinocerebellar-ataxia-type-36" class="level1">
<h1>Spinocerebellar Ataxia type 36</h1>
<p>Spinocerebellar Ataxia type 36 (<a href="https://pubmed.ncbi.nlm.nih.gov/25593102/">SCA36</a>) is a very rare neurological disorder caused by a hexanucleotide repeat expansion in the <em>NOP56</em> gene, with fewer than 200 patients reported in the literature. I optimised a repeat-primed PCR protocol and validated it on a cohort of samples tested via whole genome sequencing with the <a href="https://pubmed.ncbi.nlm.nih.gov/31134279/">ExpansionHunter</a> algorithm. This diagnostic service is, to my knowledge, the first in the world for this condition.</p>
<p><u><strong>Project Overview</strong></u></p>
<p><strong>Outcome:</strong> diagnostic service</p>
<p><strong>Timing:</strong> Nov 2022-July 2023</p>
<p><strong>Location:</strong> University College London Hospital Neurogenetics Department</p>
<p><strong>Technique:</strong> repeat-primed PCR and flanking PCR</p>
<p><u><strong>Skills I learned</strong></u></p>
<ul>
<li>How to design repeat-primed PCR primers</li>
<li>How to optimise repeat-primed PCRs</li>
<li>How to integrate a literature review into a validation document</li>
</ul>
</section>
<section id="rfc1-disorder-testing" class="level1">
<h1><em>RFC1</em>-Disorder Testing</h1>
<p><em>RFC1</em>-disorder is a neurological disorder caused by repeat expansions in the <em>RFC1</em> gene, and was first <a href="https://pubmed.ncbi.nlm.nih.gov/30926972/">reported in 2019</a> by Dr Andrea Cortese’s research group at the UCLH Institute for Neurology. For this project I developed a diagnostic service for <em>RFC1</em> expansions by modifying the available research protocol, and successfully added <em>RFC1</em> testing to the <a href="https://www.england.nhs.uk/publication/national-genomic-test-directories/">NHS National Genomic Test Directory</a>.</p>
<p><u><strong>Project Overview</strong></u></p>
<p><strong>Outcome:</strong> diagnostic service</p>
<p><strong>Timing:</strong> Jan 2022-July 2023</p>
<p><strong>Location:</strong> University College London Hospital Neurogenetics Department</p>
<p><strong>Technique:</strong> multiple repeat-primed PCRs and flanking PCR</p>
<p><u><strong>Skills I learned</strong></u></p>
<ul>
<li>How to fully optimise a flanking PCR to detect large amplicons</li>
<li>How to apply to the National Genomic Test Directory for new test indication</li>
<li>How to integrate multiple PCRs into a single workflow for increased efficiency and cost-effectiveness</li>
</ul>
</section>
<section id="mosaicism-detection-for-rare-skin-disorders" class="level1">
<h1>Mosaicism Detection for Rare Skin Disorders</h1>
<p>The aim of this project was to confirm mosaic sequence variants for rare skin disorders using droplet digital PCR (ddPCR). The main challenges were that the variants were sometimes present in 1% of the sample or lower, the DNA quantities from skin biopsies were limited, and many of the variants were only found in a single patient. The solution was to use R to collate all the ddPCR data for multiple assays and establish uniform thresholds for background normal variation. The technique was then brought into diagnostic service, and over 100 reports were issued in the first 6 months.</p>
<p><u><strong>Project Overview</strong></u></p>
<p><strong>Outcome:</strong> diagnostic service</p>
<p><strong>Timing:</strong> Oct 2021-March 2022</p>
<p><strong>Location:</strong> North London Genomic Laboratory Hub</p>
<p><strong>Technique:</strong> ddPCR</p>
<p><u><strong>Skills I learned</strong></u></p>
<ul>
<li>How to set data-driven analysis thresholds for ddPCR</li>
<li>How to use RMarkdown to generate validation plots and tables</li>
<li>How to automatically audit clinical services using RMarkdown scripts</li>
</ul>
</section>
<section id="non-invasive-prenatal-diagnosis-for-sickle-cell-disease-and-rare-genetic-conditions" class="level1">
<h1>Non-Invasive Prenatal Diagnosis for Sickle Cell Disease and Rare Genetic Conditions</h1>
<p>In this project I developed a ddPCR test for sickle cell disease which could diagnose the condition prenatally using only a maternal blood sample. The project was then expanded to developing bespoke ddPCR assays for rare genetic conditions with different inheritance patterns. The correct prediction rate was optimised to 98% in comparison with invasive testing, but the presence of 2 incorrect results prevented the implementation of the method as a diagnostic service, and we <a href="https://pubmed.ncbi.nlm.nih.gov/36760169/">published our findings</a> in the journal Prenatal Diagnosis.</p>
<p><u><strong>Project Overview</strong></u></p>
<p><strong>Outcome:</strong> publication</p>
<p><strong>Timing:</strong> September 2019-January 2022</p>
<p><strong>Location:</strong> North London Genomic Laboratory Hub</p>
<p><strong>Technique:</strong> droplet digital PCR</p>
<p><strong>Codebase:</strong> <a href="https://github.com/joeshaw824/ddpcr_nipd">R and JAGS</a></p>
<p><u><strong>Skills I learned</strong></u></p>
<ul>
<li>How to write an original research paper</li>
<li>How to write code in R using tidyverse packages</li>
<li>How to optimise ddPCR assays</li>
</ul>
</section>
<section id="myotonic-dystrophy-type-1" class="level1">
<h1>Myotonic Dystrophy Type 1</h1>
<p>This was my first validation project, which I began as a trainee clinical scientist. Myotonic dystrophy type 1 is caused by a triplet repeat expansion in the <em>DMPK</em> gene, and testing for this condition is a core requirement for genetics laboratories in England. I was asked to validate the test at the North London Genomic Laboratory Hub as part of my clinical scientist training. The validation successfully passed external evaluation by the United Kingdom Accreditation Service (UKAS) and went into diagnostic service.</p>
<p><u><strong>Project Overview</strong></u></p>
<p><strong>Outcome:</strong> diagnostic service</p>
<p><strong>Timing:</strong> July 2018-Jan 2020</p>
<p><strong>Location:</strong> North London Genomic Laboratory Hub</p>
<p><strong>Technique:</strong> repeat-primed PCR</p>
<p><u><strong>Skills I learned</strong></u></p>
<ul>
<li>How to write a validation document to the standards of ISO:15189</li>
<li>How to calculate mobility shifts in capillary electrophoresis using GeneMarker</li>
<li>How to write standard operating procedures for wet lab work and analysis</li>
</ul>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">© Copyright 2024, Joe Shaw</div>   
    <div class="nav-footer-center">
      &nbsp;
    </div>
    <div class="nav-footer-right">Built with <a href="https://quarto.org/">Quarto</a></div>
  </div>
</footer>



</body></html>